### PR TITLE
9.0 fix saas sysadmin mailgun try block

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: saas_sysadmin_mailgun/models/saas_sysadmin_mailgun.py [hunk 1]
```diff
@@ -64,15 +64,20 @@ def _create_new_database(self, **kw):
...
-        client_obj.upgrade(payload={'configure_outgoing_mail': [new_domain_info['domain']],
+            new_domain_info = simplejson.loads(mailgun_res.text)
+            ir_params = self.env['ir.config_parameter']
+            api_key = ir_params.get_param('saas_mailgun.saas_mailgun_api_key')
+            client_obj.upgrade(payload={'configure_outgoing_mail': [new_domain_info['domain']],
                                     'params': [{'key': 'mailgun.apikey', 'value': api_key, 'hidden': True},
                                            {'key': 'mail.catchall.domain', 'value': client_obj.mail_domain, 'hidden': True}]})
+            client_obj._domain_verification_and_dns_route53(new_domain_info)
+        except Exception as e:
+            _logger.exception('Error in creating Mailgun domain')
```
Comment (yelizariev) [inline saas_sysadmin_mailgun/models/saas_sysadmin_mailgun.py]: _logger.debug("Error during parsing preview", exc_info=True)

**Comment by iledarn** [comment]
@yelizariev , 
done

File: saas_sysadmin_mailgun/models/saas_sysadmin_mailgun.py [hunk 2]
```diff
@@ -64,15 +64,19 @@ def _create_new_database(self, **kw):
...
+            mailgun_res = client_obj._create_domain_on_mailgun()
+            client_obj._create_route_on_mailgun()
 
-        new_domain_info = simplejson.loads(mailgun_res.text)
-        client_obj._domain_verification_and_dns_route53(new_domain_info)
-
-        ir_params = self.env['ir.config_parameter']
-        api_key = ir_params.get_param('saas_mailgun.saas_mailgun_api_key')
-        client_obj.upgrade(payload={'configure_outgoing_mail': [new_domain_info['domain']],
+            new_domain_info = simplejson.loads(mailgun_res.text)
```
Comment (yelizariev) [inline saas_sysadmin_mailgun/models/saas_sysadmin_mailgun.py:71]: put this and below after try block

**Comment by iledarn** [comment]
@yelizariev ,
checked

**Pull request merged**